README.md

```diff
@@ -19,12 +19,12 @@ limitations under the License.
 ---
 [![Build Status][ti]][tl] [![Apache License][li]][ll] [![Maven Central][mi]][ml] [![Javadoc][ji]][jl]
 
-**Apache Fluo lets users make incremental updates to large data sets stored in Apache Accumulo.**
-
-[Apache Fluo][fluo] is an open source implementation of [Percolator][percolator] (which populates
-Google's search index) for [Apache Accumulo][accumulo]. Fluo makes it possible to update the results
-of a large-scale computation, index, or analytic as new data is discovered. Check out the Fluo
-[project website][fluo] for news and general information.
+[Apache Fluo][fluo] is a distributed processing system that lets users make incremental updates to large
+data sets. With Apache Fluo, users can set up workflows that execute cross node transactions when data
+changes. These workflows enable users to continuously join new data into large existing data sets without
+reprocessing all data. Apache Fluo is an open source implementation of [Percolator][Percolator] (which
+populates Google's search index) that is built on [Apache Accumulo]. Check out the
```

Comment:
Overall, I like the new wording, but I'm still not entirely comfortable with the mentioning of Percolator in the main description. I think it's more relevant in a longer "background" or "history" page, rather than the blurb we share to describe the project.

The reason I'm not comfortable is two-fold:

1. It seems to imply a relationship to Percolator (and Google), that does not exist. Although Fluo is inspired by the paper which describes Percolator, it is not affiliated with it in any way.
2. It makes specific claims about the nature and state of Percolator which may or may not be true, which this project does not have direct knowledge of, and can only make inferences about, and which may only be temporally true.

I would prefer it not be mentioned in the main blurb at all, or if it is, it be transformed to something like `Apache Fluo is built on Apache Accumulo, and is inspired by Google's published description of their own Percolator, which they describe as being used to populate Google's search index.`

Reply:
I agree. I think references to Google/Percolator should be removed from the README.md & main page of website. To provide users with background and give credit to paper for design, we should reference Percolator in the [design docs](https://fluo.apache.org/docs/fluo/1.2/getting-started/design). I will create another PR.